h2o-r/h2o-package/R/automl.R

```diff
@@ -157,6 +159,15 @@ h2o.automl <- function(x, y, training_frame,
     build_control$project_name <- project_name
   }
   
+  # Update build_control with nfolds
+  if (nfolds < 0) {
+    stop("nfolds cannot be negative. Use nfolds >=2 if you want cross-valiated metrics and Stacked Ensembles or use nfolds = 0 to disable.")
+  }
+  if (nfolds == 1) {
+    stop("nfolds = 1 is an invalid value. Use nfolds >=2 if you want cross-valiated metrics and Stacked Ensembles or use nfolds = 0 to disable.")
+  }
```

Comment:
Same as my comment on Py API. Just need to check `nfolds` >=2 and report back what was passed into the function in the exception.

Reply:
Yay for spaces! }notthis{

Reply:
@navdeep-G I think you're forgetting about the valid `nfolds = 0` option.  That's why I broke it up into two separate checks.

Reply:
Agreed @rpeck. Its the simple things :)

Reply:
@ledell Yes, but the check can just be done in one swoop, i.e., `nfolds >=2 or nfolds == 0`.